Bundle/TwigBundle/DependencyInjection/VictoireTwigExtension.php

```diff
@@ -28,5 +28,8 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter(
             'victoire_twig.responsive', $config['responsive']
         );
+
+        $twigConfig['globals']['victoire_twig_responsive'] = $container->getParameter('victoire_twig.responsive');
```

Comment:
instead of `getParameter`, you should just use `%victoire_twig.responsive%` and let Symfony resolve it